translations/es.json

```diff
@@ -152,7 +152,7 @@
   "label.view_source": "HTML",
   "label.open_in_lms": "Ver en LMS",
   "label.continue": "Continuar",
-  "label.content_loading_msg": "Se está revisando el contenido del curso.",
+  "label.content_loading_msg": "Se está revisando el contenido del curso. Este escaneado occure automáticamente cada vez la herramienta carga. Puede dura minutos dependiendo del contenido del curso. No se puede arreglar o resolver problemas hasta el escaneado ha completado.",
```

Comment:
It can be understood but it's a little broken. 

"UDOIT está revisando el contenido del curso. Este escaneado ocurre automáticamente cada vez que la herramienta se utiliza. Puede tardar varios minutos dependiendo del contenido del curso. No se pueden arreglar ni resolver problemas hasta que el escaneado se haya completado." would be a more direct and fitting translation!

Reply:
Great, thanks! I'll make the changes. Been a while since I took Spanish, looks like I missed several things (a preposition, wrong conjunction, wrong conjugation) and had some awkward diction! I appreciate the careful revision.

Reply:
Overall it was pretty good and it would be understood by any natural spanish speaker, but the lack of some conjuctions threw me off. Glad I could be of help!